ament_cmake_ros/ament_cmake_ros-extras.cmake.in

```diff
@@ -15,3 +15,4 @@
 # copied from ament_cmake_ros/ament_cmake_ros-extras.cmake
```

Comment:
This line should still say "generated from ... .cmake.in".

Reply:
sorry @dirk-thomas, I should have caught that myself. da58e48e2375dea12a117f7169c6d1efccc11d58